android/BOINC/app/src/main/java/edu/berkeley/boinc/client/DeviceStatus.java

```diff
@@ -118,11 +118,8 @@ private Boolean determineUserActive() throws Exception {
 		
 		if(telStatus != TelephonyManager.CALL_STATE_IDLE) {
 			newUserActive = true;
-		} else if(screenOn && appPrefs.getSuspendWhenScreenOn() && !appPrefs.getStationaryDeviceMode()) {
-			newUserActive = true;
-		} else {
-			newUserActive = false;
-		}
+		} else
+            newUserActive = screenOn && appPrefs.getSuspendWhenScreenOn() && !appPrefs.getStationaryDeviceMode();
```

Comment:
Please try to be consistent with your use of curly braces. The `if` clauses uses them while the `else` clause doesn't (and line 122 isn't correctly indented making it even more confusing).

Reply:
I think here we should either combine `telStatus != TelephonyManager.CALL_STATE_IDLE` with other statements or just left this if-else block without change

Reply:
Fixed.
@AenBleidd What do you imagine?

Reply:
Something like this:

`newUserActive = (telStatus != TelephonyManager.CALL_STATE_IDLE || (screenOn && appPrefs.getSuspendWhenScreenOn() && !appPrefs.getStationaryDeviceMode()));`

Reply:
But either @adamradocz or @brevilo find this too complicated to understand, than you initial implementation is ok too

Reply:
I think the original is easier to comprehend as boolean algebra isn't everybody's strong suit and just I don't see the benefit of squeezing that additional `or` clause in front of the existing clause. IMHO, it just increases complexity.